packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -154,24 +158,73 @@ async function generate(requestHost) {
     XLSX.utils.book_append_sheet(workbook, sheet2, 'Project Summaries');
 
     return {
+        periodId,
         filename: `audit report ${moment().format('yy-MM-DD')}.xlsx`,
         outputWorkBook: XLSX.write(workbook, { bookType: 'xlsx', type: 'buffer' }),
     };
 }
 
+function getS3Client() {
+    /*
+        TODO: Move this client generation to a separate module that handles creating all AWS clients.
+        Captured in ticket here: https://github.com/usdigitalresponse/usdr-gost/issues/1161
+    */
+
+    let s3;
+    if (process.env.LOCALSTACK_HOSTNAME) {
+        /*
+            1. Make sure the local environment has awslocal installed.
+            2. Use the commands to create a bucket to test with.
+                - awslocal s3api create-bucket --bucket arpa-audit-reports --region us-west-2 --create-bucket-configuration '{"LocationConstraint": "us-west-2"}'
+            3. Access bucket resource metadata through the following URL.
+                - awslocal s3api list-buckets
+                - awslocal s3api list-objects --bucket arpa-audit-reports
+        */
+        console.log('------------ USING LOCALSTACK ------------');
+        const endpoint = new AWS.Endpoint(`http://localstack:${process.env.EDGE_PORT}`);
```

Comment:
The "localstack" hostname here should be replaced with `${process.env.LOCALSTACK_HOSTNAME}` as well. Additionally, `$EDGE_PORT` might not always be set, so we _should_ provide a fallback value of `4566` when `process.env.EDGE_PORT` is null. 

```suggestion
        const endpoint = new AWS.Endpoint(`http://${process.env.LOCALSTACK_HOSTNAME}:${process.env.EDGE_PORT | 4566}`);
```

Reply:
Done 👍  I made those changes but got lost when I committed incorrectly. Should be fixed now.